### PR TITLE
Removing Relative imports

### PR DESCRIPTION
**Pull request by mgoldbas**
- Removing relative imports 
- Sourcing queue from multiprocessing module

### CODE REVIEW

File: tests/airflow.cfg [hunk 1]
```diff
@@ -59,7 +59,7 @@ executor = SequentialExecutor
 # The SqlAlchemy connection string to the metadata database.
 # SqlAlchemy supports many different database engine, more information
 # their website
-sql_alchemy_conn = sqlite:////c/Code/repos/KubernetesJobOperator/tests/airflow.db
+sql_alchemy_conn = sqlite:////Users/mgoldbas/PycharmProjects/KubernetesJobOperator/tests/airflow.db
```
Comment (mgoldbas) [inline tests/airflow.cfg]: we need a way to resolve this

Comment (LamaAni) [reply tests/airflow.cfg]: hum... ok.

Comment (LamaAni) [reply tests/airflow.cfg]: Fixed. See start script

**Comment by LamaAni** [comment]
Was it tested on both package install and local airflow?

**Review by LamaAni — APPROVED** [review]

**Pull request merged**